clerk_test.go

```diff
@@ -155,10 +158,10 @@ func TestBackendCall_RequestHeaders(t *testing.T) {
 		// The client sets the Authorization header correctly.
 		assert.Equal(t, fmt.Sprintf("Bearer %s", secretKey), r.Header.Get("Authorization"))
 		// The client sets the User-Agent header.
-		assert.Equal(t, "Clerk/v1 SDK-Go/v2.0.0", r.Header.Get("User-Agent"))
+		assert.Equal(t, fmt.Sprintf("Clerk/v1 SDK-Go/%s (clerk.com)", sdkVersion), r.Header.Get("User-Agent"))
```

Comment:
💡 Perhaps adding a custom string to aid in tracing the request here instead? Just to serve as a usage example.

Reply:
Just a random-looking identifier maybe?